# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../rt2570/destdir.patch
# Copyright (C) 2006 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- rt61-1.1.0-b1/Module/Makefile.orig	2006-10-02 14:58:12.000000000 +0200
+++ rt61-1.1.0-b1/Module/Makefile	2006-10-02 15:03:48.000000000 +0200
@@ -76,6 +76,12 @@
  KERNEL_OUTPUT :=
 endif
 
+ifdef DESTDIR
+ INSTALL_MOD_PATH := INSTALL_MOD_PATH=$(DESTDIR)
+else
+ INSTALL_MOD_PATH :=
+endif
+
 src ?= .
 obj ?= .
 
@@ -92,7 +98,7 @@
 	$(LD) $(EXTRA_LDFLAGS) -r -o $@ $($(MODULE_NAME)-objs)
 endif
 
-KBUILD_PARAMS := -C $(KERNEL_SOURCES) SUBDIRS=$(PWD) $(KERNEL_OUTPUT)
+KBUILD_PARAMS := -C $(KERNEL_SOURCES) SUBDIRS=$(PWD) $(KERNEL_OUTPUT) $(INSTALL_MOD_PATH)
 
 module:
 	@$(MAKE) $(KBUILD_PARAMS) modules; \
